### PR TITLE
Multiple complex fields sharing the same parent causes content loss

### PR DESCRIPTION
**Pull request by kylegibson**
If a paragraph contains two or more complex fields, content disappears.

### CODE REVIEW

File: tests/export/html/test_field_code.py [hunk 1]
```diff
@@ -12,6 +12,86 @@
...
+                    <fldChar fldCharType="begin"/>
+                </r>
+                <r>
+                    <instrText> FOOBAR baz</instrText>
+                </r>
+                <r>
+                    <fldChar fldCharType="separate"/>
+                </r>
+                <r>
+                    <t>BBB</t>
```
Comment (michaelgzoller) [inline tests/export/html/test_field_code.py]: So the behavior before was BBB would disappear?

**Comment by michaelgzoller** [comment]
What happens if multiple instruction fields are nested? Is that even possible?

**Comment by michaelgzoller** [comment]
:+1:

File: tests/export/html/test_field_code.py (hunk 1 above)
Comment (kylegibson) [inline tests/export/html/test_field_code.py]: No, actually, CCC would disappear.

**Comment by kylegibson** [comment]
> What happens if multiple instruction fields are nested? Is that even possible?

Everything breaks. It's only possible manually. The spec doesn't allow that.

**Pull request merged**